src/app/(proper_react)/(redesign)/(authenticated)/admin/dev/actions.tsx

```diff
@@ -17,18 +17,21 @@ import {
 import updateDataBrokerScanProfile from "../../../../../functions/server/updateDataBrokerScanProfile";
 import { getAllScansForProfile } from "../../../../../../db/tables/onerep_scans";
 import { refreshStoredScanResults } from "../../../../../functions/server/refreshStoredScanResults";
+import { isMozMail } from "../../../../../functions/universal/isMozMail";
 
 export async function lookupFxaUid(emailHash: string) {
   const session = await getServerSession();
-  if (
-    !session?.user?.email ||
-    !isAdmin(session.user.email) ||
-    process.env.APP_ENV === "production"
-  ) {
+  if (!session?.user?.email || !isAdmin(session.user.email)) {
     return notFound();
   }
 
   const subscriber = await getSubscribersByHashes([emailHash]);
+  if (
+    process.env.APP_ENV === "production" &&
```

Comment:
As suggested elsewhere:

```suggestion
    // On production, only allow looking up Mozilla email addresses
    process.env.APP_ENV !== "stage" && process.env.APP_ENV !== "local" &&
```

Reply:
Update in [7b598da](https://github.com/mozilla/blurts-server/pull/5850/commits/7b598dab3a98bfcab7e7fe0d43a4186da03ede98).